### ISSUE TITLE
Image(BufferedImage) --> Image(Mat) is slow and broken?

### ISSUE THREAD

**Issue by AlexDBlack** [KonduitAI/konduit-serving#424]
For large images it's seconds per frame, and everything has a red hue when visualized via ShowImageStep

**Comment by AlexDBlack** [comment]
Possibly related: https://github.com/KonduitAI/konduit-serving/issues/426

https://stackoverflow.com/a/3432532
>AFAIK, regarding the "pinkish tint", Java saves the JPEG as ARGB (still with transparency information). Most viewers, when opening, assume the four channels must correspond to a CMYK (not ARGB) and thus the red tint.

**Comment by AlexDBlack** [comment]
This PR fixes the tint issue: https://github.com/KonduitAI/konduit-serving/pull/427

However it's still slow (though perhaps less slow than in was with the alpha channel...)

**Comment by saudet** [comment]
Java 2D in general is slow. It should be used only for legacy support or for where performance isn't a problem.

**Comment by AlexDBlack** [comment]
Yeah, it's more for interop, all image steps in KS use javacv. We're using BufferedImage as the source of some generated data in a demo. What we're doing there is not complex (combining a couple of images that have alpha channels). It could simply be easier to rewrite that part in javacv if it proves to be a bottleneck in practice than worrying about optimizing this right now.

More generally for the conversion, it should be possible (at least in some circumstances) to pass or adapt the DataBuffer from the BufferedImage Raster to javacv; currently this is using the (not exactly performant/elegant/well designed) approach of using png as an intermediate format.
